plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcDeleteDeltaPageSource.java

```diff
@@ -139,7 +137,7 @@ private OrcDeleteDeltaPageSource(
         this.stats = requireNonNull(stats, "stats is null");
         this.orcDataSource = requireNonNull(orcDataSource, "orcDataSource is null");
 
-        verifyAcidSchema(reader, path);
+        verifyAcidSchema(reader, new Path(path));
```

Comment:
You could migrate `verifyAcidSchema` pretty easily as well. Looks like it just uses the path in error messages

Reply:
I actually tried that and it turned out it is not that simple - I would also have to make changes in `OrcPageSourceFactory` to `verifyFileHasColumnNames` and some more places - there would be a lot of changes and this is not really related.